src/buildercore/cfngen.py

```diff
@@ -83,9 +83,17 @@ def build_context(pname, **more_context): # pylint: disable=too-many-locals
     # post-processing
     if 'rds' in context['project']['aws']:
         default_rds_dbname = slugify(stackname, separator="")
+
+        # used to give mysql a range of valid ip addresses to connect from
+        net, bits = context['project']['aws']['subnet-cidr'].split('/', 1)
+        # todo: determine mask based on bits
+        mask = '255.255.255.0'
+        netmask = "%s/%s" % (net, mask) # ll: 10.0.2.0/255.255.255.0
+
         # alpha-numeric only
         # TODO: investigate possibility of ambiguous RDS naming here
         context.update({
+            'netmask': netmask,
```

Comment:
it should be possible to extend or add a test in https://github.com/elifesciences/builder/blob/master/src/tests/test_buildercore_trop.py, which tests `context` fields before using them for the CloudFormation templates